calculator/script.js

```diff
@@ -0,0 +1,28 @@
+window.onload = function() {
+
+    var calc = document.getElementById('calc');
+    var operations = {
+        "+": function(a, b) {
+            return a + b;
+        },
+        "-": function(a, b) {
+            return a - b;
+        },
+        "*": function(a, b) {
+            return a * b;
+        },
+        "/": function(a, b) {
+            return a / b;
+        },
+        "%": function(a, b) {
+            return Math.floor(a / b);
```

Comment:
Math.floor(-5 / 2) = -3 but it should be -2
there is another func in Math, or you can combine Math.floor + Math.ceil